### ISSUE TITLE
Installation issues

### ISSUE THREAD

**Issue by MaximilianAlgehed** [adamwalker/haskell_cudd#13]
Hi,

I'm trying to install the library.
After installing cudd using
```
./configure --enable-dddmp
make
make install
```
And in a fresh cabal sandbox doing
```
cabal install c2hs
cabal install cudd
```
I get the following error
```
In file included from dist/dist-sandbox-242b4ce8/build/Cudd/File.chs.h:2:0:
/usr/local/include/dddmp.h:40:18: fatal error: util.h: No such file or directory
 #include "util.h"
```

Any clues as to what is going on here?

**Comment by adamwalker** [comment]
That's strange. Is util.h in /usr/local/include?

**Comment by MaximilianAlgehed** [comment]
Nope, not by default. What you need to do is add `util.h` and `mtr.h` to `/usr/local/include/` after installing `CUDD` (it doesn't do it by default...). You also need to build `CUDD` with `--enable-shared`. Furthermore, on my machine I had to add `/usr/local/lib` to `$LD_LIBRARY_PATH`.

Perhaps it's a good idea to update the install instructions to take this in to account (or maybe it's just my machine being rubbish)?

**Comment by adamwalker** [comment]
Yeah, I actually remember having to copy util.h over now. Any ideas why that doesnt happen automatically? 
I'll update the installation instructions. Thanks for the bug report.

**Comment by MaximilianAlgehed** [comment]
Well, I suppose it's an error on the cudd side. Not much we can do about it.

**Comment by jwaldmann** [comment]
see #6
